perfkitbenchmarker/data/__init__.py

```diff
@@ -28,9 +28,10 @@
 
 import pkg_resources
 
+from perfkitbenchmarker import context_util
 from perfkitbenchmarker import flags
 
-FLAGS = flags.FLAGS
+FLAGS = context_util.FLAGS
```

Comment:
No, I don't think I have anything good. Attaching flags to the BenchmarkSpec *and* changing references from module-level flags to benchmark spec flags instances would help, but would require passing flag values through to packages, etc.